### ISSUE TITLE
Unable to click link in Microsoft Powerpoint for Office 365

### ISSUE THREAD

**Issue by math65** [nvaccess/nvda#11258]
### Steps to reproduce:

When a presentation is displayed, NVDA does not recognize that it is a link, when you press Enter it goes to the next slide, and NVDA + enter does not work.

### Actual behavior:

* Create a presentation
* Add a slide
* Add a link in the slide with a description (for example, access the site)
* start the presentation by pressing F5
* Place the focus on the link and press Enter

### Expected behavior:

The link does not activate, the next slide is displayed. When you press NVDA + Enter, it doesn't work.

#### NVDA installed/portable/running from source:

Installed

#### NVDA version:

2020.1

#### Windows version:

Windows 10, version 2004

#### Name and version of other software in use when reproducing the issue:

Microsoft Powerpoint for microsoft 365

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No

#### If addons are disabled, is your problem still occuring?

Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?

Yes

**Comment by Qchristensen** [comment]
This is a duplicate of issue #7288

**Comment by Qchristensen** [comment]
Just a clarification @math65 - Is this issue that when you TAB to an issue and press ENTER nothing happens? or that you want to down arrow to a link and press ENTER to activate it?

You should be able to press TAB to move to a link and ENTER to activate it.  The problem is (as per issue #7288) that nothing is reported when you press TAB.  If #7288 gets resolved, would that resolve your issue (by giving you an accessible way to activate a link in a presentation with appropriate feedback on where the focus is)?  Or would you still like to be able to arrow to a link and press enter to activate it?

**Comment by math65** [comment]
@Qchristensen Yes, if the other issue was resolved, it would be perfect!
If I can access the link with Tab it's good! :)

**Comment by Qchristensen** [comment]
Ok, I was perhaps a bit hasty in closing this issue, but we'll leave it closed and concentrate on rectifying the other issue.  So in short, yes you can access links with TAB, just be aware that at this stage they don't speak anything when you do press TAB, but if you press TAB and then ENTER it will open the link in your browser and that will read normally.

**Comment by math65** [comment]
Ok, thank so much !

I hope it can be resolved! Because it's been a long time since it was opened;)
